docs/http-response-status.rst

```diff
@@ -1,25 +1,27 @@
 HTTP Response Status
 ====================
 
-List of HTTP status codes and respective descriptions.
+List of HTTP status codes and possible messages.
 
 2xx success
 -----------
 
-* ``200 OK`` - standard response for successful GET or POST requests
+``200 OK`` - standard response for successful GET or POST requests.
+
+.. note:: If an error occurs during a data-extraction request, the result will be a broken GeoJSON containing the error JSON object at the end. In this case, the ohsome API will give back a 200 response status.
 
 4xx client errors
 -----------------
 
-* ``400 Bad Request`` - the ohsome API cannot or will not process the request due to a client error (e.g. malformed request syntax)
-* ``401 Unauthorized`` - the client does not have valid authentication credentials for the target resource
-* ``404 Not Found``-  the requested resource could not be found
-* ``405 Method not allowed`` - the requested method is not supported (e.g. a POST request for resources which accept only GET requests)
-* ``413 Payload Too Large`` - the request is larger than the ohsome API is willing or able to process
+``400 Bad Request`` - the ohsome API cannot process the request due to a client error. This can occur because a parameter is not given in the correct format, mandatory parameters are not present, unknown or not valid parameters are present, parameters are not unique, malformed GeoJSON, unsopported content-type reader, wrong parameter values or other malformed requests.
+
+``404 Not Found``-  the requested resource could not be found because the given boundary or time are not within the underlying osh-data.
+
+``413 Payload Too Large`` - the request is too large in respect to the given timeout parameter.
```

Comment:
imho this could be worded a bit simpler/better? Specifically _"too large"_ means in our case that it _"took too long"_, and not all queries have a _"given timeout"_, but use the default one:

```suggestion
``413 Payload Too Large`` - the request took too long to compute.
```

Reply:
OK, but it seems to me it doesn't inform the user in respect of what it took too long. What do you think about `the request took too long to compute in respect of the given or default timeout.`?

Reply:
that should work